### DISCUSSION TITLE
AdoptOpenJDK replaced by Adoptium Temurin

### DISCUSSION THREAD

**Discussion by russellbanks** [microsoft/winget-pkgs#62877] [Q&A]
Seeing as AdoptOpenJDK has been replaced by Adoptium Temurin (and AdoptOpenJDK builds are no longer recommended), should those packages be removed or are they being kept on here as an archive?

**Comment by jedieaston** [comment]
We usually keep them up as an archive unless they aren’t being hosted anymore by the source. If they still download and work, there’s no reason to remove them. 

Once the install notes functionality is present in the stable client (https://github.com/microsoft/winget-cli/issues/607) we should add a note to them mentioning that up to date JDKs are now under a different ID.

**Reply by Trenly** [reply]
This is also another use case for optional redirects microsoft/winget-cli#1899

**Reply by denelon** [reply]
@russellbanks a recent suggestion from @Trenly points a way forward:
* https://github.com/microsoft/winget-pkgs/issues/66937#issuecomment-1190154419
